go/vt/vttablet/tabletserver/schema/engine_test.go

```diff
@@ -658,3 +661,563 @@ func AddFakeInnoDBReadRowsResult(db *fakesqldb.DB, value int) *fakesqldb.Expecte
 		fmt.Sprintf("Innodb_rows_read|%d", value),
 	))
 }
+
+func TestEngineMysqlTime(t *testing.T) {
```

Comment:
nit: add a comment about the reason for this test.

Reply:
It is testing the function called `mysqlTime`. I didn't add a comment, because it won't be useful.